kit/Kit.cpp

```diff
@@ -165,16 +165,8 @@ static void flushTraceEventRecordings();
 // socket buffer & event processing in a single, thread.
 bool pushToMainThread(LibreOfficeKitCallback cb, int type, const char *p, void *data);
 
-#if !MOBILEAPP
```

Comment:
Is this change intended ? =) seems mobile builds are failing on the branch here - though we don't so much care about mobile for 23.05 anymore I think.